packages/schemas/alterations/next-1668666590-support-multiple-connector-instances.ts

```diff
@@ -5,6 +5,8 @@ import type { AlterationScript } from '../lib/types/alteration.js';
 const alteration: AlterationScript = {
   up: async (pool) => {
     await pool.query(sql`
+      DELETE FROM connectors WHERE enabled = false;
```

Comment:
This alteration is merged into master, is not forbidden to modify, please create a new alteration script.

Reply:
Move them to a new script.